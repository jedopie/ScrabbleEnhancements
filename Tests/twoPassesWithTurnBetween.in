2
Tests/save.save
pass
place A at A0
done
place R at K0
done
place E at I0
done
pass
