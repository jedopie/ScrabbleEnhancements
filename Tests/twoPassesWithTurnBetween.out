Welcome to Scrabble!
-------------------
Menu
----
1. New Game
2. Load Game
3. Credits (Show student information)
4. Quit

> 
Enter the filename from which load a game
> 
Scrabble Game Successfully Loaded!

JOHN, it's your turn
Score for JOHN: 0
Score for JED: 1
   0   1   2   3   4   5   6   7   8   9  10  11  12  13  14
  ------------------------------------------------------------
A|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
B|   | N |   |   |   |   |   |   |   |   |   |   |   |   |   | 
C|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
D|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
E|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
F|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
G|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
H|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
I|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
J|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
K|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
L|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
M|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
N|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
O|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 

Your hand is
R-1,D-2,O-1,R-1,I-1,N-1,D-2
> 
JED, it's your turn
Score for JOHN: 0
Score for JED: 1
   0   1   2   3   4   5   6   7   8   9  10  11  12  13  14
  ------------------------------------------------------------
A|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
B|   | N |   |   |   |   |   |   |   |   |   |   |   |   |   | 
C|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
D|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
E|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
F|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
G|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
H|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
I|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
J|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
K|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
L|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
M|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
N|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
O|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 

Your hand is
R-1,A-1,B-3,I-1,T-1,O-1,E-1
> > 

JOHN, it's your turn
Score for JOHN: 0
Score for JED: 2
   0   1   2   3   4   5   6   7   8   9  10  11  12  13  14
  ------------------------------------------------------------
A| A |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
B|   | N |   |   |   |   |   |   |   |   |   |   |   |   |   | 
C|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
D|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
E|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
F|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
G|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
H|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
I|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
J|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
K|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
L|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
M|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
N|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
O|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 

Your hand is
R-1,D-2,O-1,R-1,I-1,N-1,D-2
> > 
JED, it's your turn
Score for JOHN: 1
Score for JED: 2
   0   1   2   3   4   5   6   7   8   9  10  11  12  13  14
  ------------------------------------------------------------
A| A |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
B|   | N |   |   |   |   |   |   |   |   |   |   |   |   |   | 
C|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
D|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
E|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
F|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
G|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
H|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
I|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
J|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
K| R |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
L|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
M|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
N|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
O|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 

Your hand is
R-1,B-3,I-1,T-1,O-1,E-1,L-1
> > 

JOHN, it's your turn
Score for JOHN: 1
Score for JED: 3
   0   1   2   3   4   5   6   7   8   9  10  11  12  13  14
  ------------------------------------------------------------
A| A |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
B|   | N |   |   |   |   |   |   |   |   |   |   |   |   |   | 
C|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
D|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
E|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
F|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
G|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
H|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
I| E |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
J|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
K| R |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
L|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
M|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
N|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
O|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 

Your hand is
D-2,O-1,R-1,I-1,N-1,D-2,O-1
> 
JED, it's your turn
Score for JOHN: 1
Score for JED: 3
   0   1   2   3   4   5   6   7   8   9  10  11  12  13  14
  ------------------------------------------------------------
A| A |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
B|   | N |   |   |   |   |   |   |   |   |   |   |   |   |   | 
C|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
D|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
E|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
F|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
G|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
H|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
I| E |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
J|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
K| R |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
L|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
M|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
N|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
O|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 

Your hand is
R-1,B-3,I-1,T-1,O-1,L-1,F-4
> 